chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -124,37 +131,21 @@ object CBPurchase {
             )
         }
     }
-    @JvmStatic
-    @Throws(InvalidRequestException::class, OperationFailedException::class)
-    fun queryPurchaseHistory() {
-        try {
-            billingClientManager?.queryPurchaseHistory()
-        }catch (exp: Exception){
-            Log.i(javaClass.simpleName, "Exception in validateReceipt() :"+exp.message)
-            ChargebeeResult.Error(
-                exp = CBException(
-                    error = ErrorDetail(
-                        exp.message
-                    )
-                )
-            )
-        }
-    }
-    @JvmStatic
-    @Throws(InvalidRequestException::class, OperationFailedException::class)
-    fun queryAllPurchases() {
-        try {
-            billingClientManager?.queryAllPurchases()
-        }catch (exp: Exception){
-            Log.i(javaClass.simpleName, "Exception in validateReceipt() :"+exp.message)
-            ChargebeeResult.Error(
-                exp = CBException(
-                    error = ErrorDetail(
-                        exp.message
-                    )
-                )
-            )
-        }
```

Comment:
@cb-amutha Since we are removing these public methods, are we sure that these are not being used by the customers? If not can we deprecate them with appropriate warnings instead and remove it in future releases?